com.rlabrecque.steamworks.net/Runtime/CallbackDispatcher.cs

```diff
@@ -69,12 +73,54 @@ internal static void Shutdown() {
 				if (m_initCount == 0) {
 					UnregisterAll();
 					Marshal.FreeHGlobal(m_pCallbackMsg);
+					Marshal.FreeHGlobal(s_pCallResultBuffer);
 					m_pCallbackMsg = IntPtr.Zero;
+					s_pCallResultBuffer = IntPtr.Zero;
 				}
 			}
 		}
 
-		internal static void Register(Callback cb) {
+		private static IntPtr CheckCallResultBufferSize(uint requiredBufferSize)
+        {
+            if (s_currentCallResultBufferSize >= requiredBufferSize)
+                return s_pCallResultBuffer; // buffer is enough, this happens mostly
```

Comment:
Can we wrap all of these single line if statements in { } ?

Also comment is semi-cut off

Reply:
Sure, style issues should be fixed at final stage. Now I want to fix code design first.

Reply:
Now all re-formatted. I think add a .editorconfig will help to enforce style issues.